xdp-dump/Makefile

```diff
@@ -6,5 +6,6 @@ TEST_FILE := test-xdpdump.sh
 
 LIB_DIR   = ../lib
 USER_LIBS = -lpcap
+USER_CFLAGS = -DXDPDUMP_VERSION=\"$(LIBXDP_VERSION)\"
```

Comment:
After re-thinking this, I will change this to an XDPDUMP specific version in the Makefile, rather than use the libxdp version. Let me know if you still want to keep the USER_CFLAGS as it might be handy for other use cases?

Reply:
If a tool needs a particular CFLAG, can't the Makefile in that directory just do 'CFLAGS+=foo' ?

As for the libbpf version detection, I started hacking on it yesterday, but realised that we may be better off doing what configure scripts usually do: detect the feature itself, instead of matching on versions.

I.e., we can have configure try to compile a stub program with the perf buffer flush call in it, and if that succeeds, we set HAVE_PERF_FLUSH or some-such... I think this will be more maintainable in the long run :)

Reply:
The 'CFLAGS+=foo' does not work as common.mk is included below the use definitions. Maybe common.mk needs changing to make this work. For now I'll remove this patch and make it direct version reference.

Reply:
Well just add the += underneath the include statement?